packages/plugin-conversation/src/normalizer.js

```diff
@@ -6,23 +6,19 @@
 
 import {patterns} from '@ciscospark/common';
 import {SparkPlugin} from '@ciscospark/spark-core';
-import {filter} from '@ciscospark/helper-html';
-import {isArray, isString} from 'lodash';
+import {isArray, isFunction, isString} from 'lodash';
 import S from 'string';
 
 const Normalizer = SparkPlugin.extend({
```

Comment:
I like the new structure where we have the inbound and outbound normalizers extending normalizer. With these changes, however, normalizer is now a sort of abstract base class because it needs `filter()` to be implemented in order to be used. Should we document this somehow? Do we have a documentation pattern for labeling a plugin like this as needing to be extended?